### PR TITLE
Jose/ListView : Reviewed additions to ListView

### PR DESCRIPTION
**Pull request by Josetf2396**

### CODE REVIEW

**Comment by Josetf2396** [comment]
image

**Comment by Josetf2396** [comment]
image

File: src/components/listView/ListView.styles.js [hunk 1]
```diff
@@ -1,11 +1,9 @@
 import {StyleSheet} from 'react-native';
+import {size} from '../../theme';
```
Comment (Nap1400) [inline src/components/listView/ListView.styles.js:2]: Will this include the amount of padding?

**Review by Nap1400 — APPROVED** [review]

File: src/components/listView/ListView.view.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ export const ListViewInfo = ({data, ...rest}) => {
   return (
     <View style={styles.container}>
       <FlatList
+        //R
```
Comment (raynesmike) [inline src/components/listView/ListView.view.js:11]: remove unused comment

File: src/components/listView/ListView.styles.js (hunk 1 above)
Comment (Josetf2396) [reply src/components/listView/ListView.styles.js:2]: yes

File: src/components/listView/ListView.view.js (hunk 2 above)
Comment (Josetf2396) [reply src/components/listView/ListView.view.js]: done

**Review by leo-he-1 — APPROVED** [review]

**Comment by Dulaya** [comment]
Do we still need this PR? Otherwise, we should delete it.

**Pull request closed without merging**